src/Storage/AuthorizationCodeStorage.php

```diff
@@ -115,4 +115,28 @@ public function cleanupOldCodes() {
 			}
 		}
 	}
+
+	public static function uninstall() {
+		global $wpdb;
+
+		// Following query is only possible via a direct query since meta_key is not a fixed string
+		// and since it only runs at uninstall, we don't need it cached.
+		// phpcs:ignore WordPress.DB.DirectDatabaseQuery.DirectQuery,WordPress.DB.DirectDatabaseQuery.NoCaching
+		$data = $wpdb->get_results(
+			$wpdb->prepare(
+				"SELECT user_id, meta_key FROM $wpdb->usermeta WHERE meta_key LIKE %s",
+				'oidc_expires_%',
+			)
+		);
```

Comment:
Is there a reason we don't just delete this right away?

```suggestion
		$wpdb->query( "DELETE FROM $wpdb->usermeta WHERE meta_key LIKE 'oidc_expires_%'" );,
```

There are a bunch of filters being called but can you imagine anything that would happen with those during an uninstall?

Reply:
For proper updation of cache, we must use the delete_* abstractions

Reply:
In my opinion, this being an uninstall script, it's easier for anyone with large installs to have this script run outside of usual "delete plugin" flow in async manner, if they wanted.

Default behaviour should maintain cache integrity while deleting using abstractions to delete, even if that's slow. Even if it times out, this script is all that one needs to run again (it being idempotent in nature).

Deleting with a direct query and then writing scripts to bring cache to a consistent state seems like an unnecessary & impractical route to take.